source/clear-linux/get-started/bare-metal-install-server/bare-metal-install-server.rst

```diff
@@ -236,154 +233,216 @@ on it, and accept the `Default partition schema`_.
+
+   .. figure:: figures/bare-metal-install-server-13.png
       :scale: 100%
-      :alt: boot partition
+      :alt: CLR_BOOT
 
-      Figure 11: boot partition
+      Figure 13: CLR_BOOT
 
-#. Navigate to :guilabel:`Add` and press :guilabel:`Enter`.
+Now follow the same process to configure remaining partitions.
 
 swap partition
 --------------
 
-#. In the :guilabel:`File System` pulldown menu, select `swap`, and
-   enter a label. We enter the minimum required size (e.g., 256M).
+#. Use the :kbd:`Up/Down` arrow to select free space.
 
-   .. figure:: figures/bare-metal-install-server-12.png
-      :scale: 100%
-      :alt: swap partition
+#. Select :guilabel:`[New]` at bottom and press :kbd:`Enter`.
 
-      Figure 12: swap partition
+#. Under :guilabel:`First sector`, press :kbd:`Enter`.
 
-#. Navigate to :guilabel:`Add` and press :guilabel:`Enter`.
+#. For :guilabel:`Size in sectors`, type 256M, and press :kbd:`Enter`.
 
-#. Next, navigate to :guilabel:`Confirm` and press :guilabel:`Enter`,
-   shown in Figure 13.
+#. Enter the hex code `8200` and press :kbd:`Enter`.
 
-   Manual partitioning is complete.
+#. In :guilabel:`Enter new partition name...`, type CLR_SWAP.
 
-   .. figure:: figures/bare-metal-install-server-13.png
-      :scale: 100%
-      :alt: Final configuration of disk partitions
+#. Press :kbd:`Enter`.
 
-      Figure 13: Final configuration of disk partitions
+   .. note::
 
-#. You may skip to the `Telemetry`_ section below.
+      |CL| allows more than one swap partition.
 
-Disk encryption
-===============
+root partition
+--------------
 
-For greater security, disk encryption is supported using LUKS for the
-any partition except `/boot` on |CL|. To encrypt the root partition, see the
-example below. Encryption is optional.
+#. Use the :kbd:`Up/Down` arrow to select free space.
 
-Encryption Passphrase
----------------------
+#. Now select :guilabel:`[New]` at bottom and press Enter.
 
-|CL| uses a single passphrase for encrypted partitions. Additional keys may
-be configured post-installation using the ``cryptsetup`` tool.
+#. Under :guilabel:`First sector`, press Enter.
 
-#. Optional: Select :guilabel:`[X] Encrypt` to encrypt the root partition,
-   as shown in Figure 14.
+#. For :guilabel:`Size in sectors`, type in desired size.
+
+   .. note::
+
+      Press :kbd:`Enter` to accept the remaining space available.
+
+#. Press Enter.
+
+#. Enter the hex code `8300` and press :kbd:`Enter`.
+
+#. In :guilabel:`Enter new partition name...`, type: CLR_ROOT.
+   The `/root` partition must be `ext[234]` or `XFS`.
+
+   .. note::
+
+      You may also append the following labels:
+
+      *  `CLR_ROOT_E`: Adds encryption; not valid for CLR_BOOT
+      *  `CLR_ROOT_F`: Formats the partition prior to use
+      *  `CLR_ROOT_E_F`: Adds encryption and formats the partition
+
+      If no file system exists, the installer will default to `VFAT(FAT32)`
+      for `/boot`, and `ext4` for all others.
```

Comment:
Maybe this could be a separate bullet point instead of being under root partition as it talks about the boot partition too.

Reply:
Point taken. Instead, I'll add a note under the /boot section indicating it cannot be encrypted. Cool?

Reply:
cool